Project.toml

```diff
@@ -62,17 +62,17 @@ DiffEqCallbacks = "2.25"
 Downloads = "1.6"
 EllipsisNotation = "1.0"
 FillArrays = "0.13.2, 1"
-ForwardDiff = "0.10.18"
-HDF5 = "0.14, 0.15, 0.16, 0.17"
+ForwardDiff = "0.10.24"
+HDF5 = "0.17"
```

Comment:
This prevents us from using older HDF5 versions, was this intentional (I guess so)

Reply:
Yes, older versions give incompatibilities with other packages. But I just checked that we can actually use HDF5.jl v0.16.10 (but not v0.16.9 and lower).

Reply:
Thanks 👍